### ISSUE TITLE
Blurring of RAW preview at "Fit to screen" zoom level

### ISSUE THREAD

**Issue by simohno** [Beep6581/RawTherapee#5673]
Hello to all,

I am new to RawTherapee, that I find an amazing software, but having opened it for the first time on an external 24" screen (version 5.8), I've found an issue which, probably, was present also on the 14" laptop screen I used till some ago. The RAW preview in Editor mode is somewhat blurred at "Fit to screen" zoom level (that for me is 24%, all the tabs open), compared to the exported JPG and opened with XNView.

Having googled a bit about this, I've found on RawPedia that one possible cause for this is the application of one or more "Lens/Geometry" tools. I tried, **with no crop or rotation, to switch off any Lens correction (No lens profile) and also any sharpening tool (2nd tab) and the new Capture sharpening feature**; the degree of blur seems a bit reduced but remains. I noticed on the other hand that setting the zoom to the rounded 25% value (exactly 1:16 subsampling from original), the two images in RT and XNV are almost identical.

What am I doing wrong? Is this normal for "strange" zoom levels as 24%? Maybe does it depend on RT's subsampling algorithm for the preview? But I would expect anyone else to have reported this too, so maybe this is not a regular behaviour...

Screenshots of the RT preview and JPG at 24% and 25% follow, in order:
RT at 24%
![1-RT 24%](https://user-images.githubusercontent.com/61553300/75463303-e8d4bd00-5985-11ea-9733-324816bfaf90.png)
XNView at 24%
![2-XNV 24%](https://user-images.githubusercontent.com/61553300/75463315-eb371700-5985-11ea-9696-98acb0d081c7.png)
RT at 25%
![3-RT 25%](https://user-images.githubusercontent.com/61553300/75463317-ec684400-5985-11ea-93e6-f27d83ca0ccc.png)
XNView at 25%
![4-XNV 25%](https://user-images.githubusercontent.com/61553300/75463321-ed997100-5985-11ea-8bee-c69e4e613c54.png)

I find no difference when viewing the two at 100%.

Thank you!

RT version: 5.8 (stable, commit 9a9e0acbf)
OS: Windows 10 64-bit version 19.09 (build 18363.657)

**Comment by Thanatomanic** [comment]
Hi!
> Maybe does it depend on RT's subsampling algorithm for the preview? 

That would be my bet. Also, some tools only work when viewed at 100% zoom. Their effects cannot be seen otherwise. Obviously the rescaled output looks different then.

**Comment by simohno** [comment]
Actually there are no "1:1" tools applied. See the PP3.

I agree that probably it relies on RT's downscaling algorithm, also because honestly every software seems to produce a slightly different result:
- Tried with Gimp, which strangely requires a 24.4% zoon level to achieve the exact same dimension on screen, and the "blurring" of the results is more close to RT than to XNV.
- Tried with Windows Photos, the preview is the sharpest, even a bit too much
- Tried opening the JPG directly with Crome, maybe the best reference (where others see our photos), and it is really, really close to XNView's at the same size.

The stair, from the sharpest: Windows Photos > XNView > Chrome > GIMP > RawTherapee.

In conclusion, I don't know if this is to be considered an issue, to check sharpness one should zoom at 100%, but I would love indeed that RT's "small" preview were a bit more crisper, like in Chrome. As a workaround I may fit the upper panel in RT to achieve 25% zoom, the result becomes nice as I expect, so that in many case I wouldn't zoom in at 100%.

So just a point to consider for future versions of RT.

Regards

PP3 in zip
[RITROVO BOLOGNA 14-15 APRILE 004.ORF.bb897cb921220bbd627ed55555474f8e.pp3.zip](https://github.com/Beep6581/RawTherapee/files/4266624/RITROVO.BOLOGNA.14-15.APRILE.004.ORF.bb897cb921220bbd627ed55555474f8e.pp3.zip)

**Comment by Beep6581** [comment]
See "Advanced", point 9:
http://rawpedia.rawtherapee.com/Getting_Started#Advanced
